### ISSUE TITLE
The extension thumbnail should have an empty ALT

### ISSUE THREAD

**Issue by neeleshb** [microsoft/vsmarketplace#744]
Type: <b>Bug</b>

WCAG 2.0 recommends that every <img> tag must have an ALT attribute. The value should be empty if the image is purely decorative. 
https://www.w3.org/TR/WCAG20-TECHS/H37.html

Extension thumbnails are decorative, so they should have empty ALT. 

VS Code version: Code 1.78.0 (252e5463d60e63238250799aef7375787f68b4ee, 2023-05-03T20:09:46.685Z)
OS version: Linux x64 6.3.11-1rodete1-amd64
Modes:
Sandboxed: No

**Comment by neeleshb** [comment]
Incorrectly filed, please ignore. Closing the issue.
